R/label_roche.R

```diff
@@ -35,3 +35,58 @@ label_roche_pvalue <- function(big.mark = ifelse(decimal.mark == ",", " ", ","),
                                ...) {
   function(x) roche_pvalue(x, big.mark = big.mark, decimal.mark = decimal.mark, ...)
 }
+
+#' Round percentages
```

Comment:
Let's document this function in the same file as the p-value formatting function.

Reply:
done

Reply:
![image](https://github.com/user-attachments/assets/6bc31919-a514-4842-9d18-c3a775614ea5)

The roxygen comments need to be updated to indicate all the functions documented in the file. This only mentions the percent, and the first functions listed at the pvalue formatters